src/Files.App/Controllers/SidebarPinnedController.cs

```diff
@@ -18,7 +18,7 @@ public class SidebarPinnedController : IJson
 	{
 		public SidebarPinnedModel Model { get; set; }
 
-		public EventHandler<NotifyCollectionChangedEventArgs> DataChanged;
+		public EventHandler<NotifyCollectionChangedEventArgs>? DataChanged;
```

Comment:
When would this be null?

Reply:
It can't be null. This property is nullable because it is used as an event and unsubscribing causes a warning if the event is not nullable.
This property is not an event because it is triggered from another class. It's not the best way to do it but that's not the purpose of this pr.

Reply:
We're trying to stay away from making properties/events nullable unless necessary. The preferred solution is to solve the underlying problem. In this case, we'd need to merge the controller with the view model.

Reply:
This change is to avoid a warning in SidebarViewModel. The controller and the model will have to be refactored but it is a big job and it is not the object of this pr. The behavior must be analyzed because the code is not clean.

Reply:
It's best to leave as it was, simply removing the warning doesn't solve the issue.

Reply:
Nope. Nullable should always be applied to events unless it has a default, non-null value

Reply:
I set this DataChanged back to non-nullable. The SidebarPinnedModel/Controller logic needs to be reviewed (I'll do it in January). The other DataChanged have been converted to an event, which makes more sense.